client/webserver/api.go

```diff
@@ -242,6 +275,14 @@ func (s *WebServer) apiAccountImport(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	r.Close = true
+	if len(form.Pass) == 0 {
+		cachedPass, err := s.getCachedPassword(r)
+		if err != nil {
+			log.Errorf("unable to get cached pw: %v", err)
```

Comment:
Error message would be nonsensical if no password has been cached and user is not utilizing the feature.